comeet/Data/Parsers/FreebusyBlockParser.swift

```diff
@@ -22,8 +22,9 @@ class FreebusyBlockParser {
             let endString = freebusyBlockDict[Constants.endKey] as? String else {
                 return nil
         }
-        guard let start = Date.fromAPIFreeBusy(stringDate: startString),
-            let end = Date.fromAPIFreeBusy(stringDate: endString),
+
+        guard let start = Date.fromISO8601(string: startString),
```

Comment:
I tested after @rrballenger1717 changes and this is working. By test I mean we are parsing correctly the response. But I'm not a 100% sure.